### DISCUSSION TITLE
有3个master节点，但是删除了Master0节点，在后期加入节点时候提示某些镜像拉取不到，怎么手动解决？

### DISCUSSION THREAD

**Discussion by sun363587351** [labring/sealos#5018] [Q&A]
有3个master节点，但是删除了Master0节点，在后期加入节点时候提示某些镜像拉取不到，怎么手动解决？

**Comment by sun363587351** [comment]
> 有3个master节点，但是删除了Master0节点，在后期加入节点时候提示某些镜像拉取不到，怎么手动解决？

目前在新master节点重启register服务解决，但是在加入步骤提示找不到kubeadm-certs  请问下sealos加入步骤过程   是不是master0一定不能挂？

**Comment by taoyang987** [comment]
你是怎么删除master0节点的,我用sealos delete --masters 删除提示我不允许删除

**Comment by taoyang987** [comment]
https://github.com/labring/sealos/discussions/4996
我研究出来了,看这个
